packages/components/src/spinner/style.scss

```diff
@@ -4,7 +4,7 @@
 	width: 18px;
 	height: 18px;
 	opacity: 0.7;
-	float: right;
+	float: none;
```

Comment:
Should I keep this as `float:none;` or just remove it altogether? @jasmussen any thoughts?

Reply:
Interesting, if the float does not appear to regress anything, definitely remove it altogether. Floats are the worst, and if there's truly no downside to removing it, out it goes.

A float left or right literally takes an item out of the flow of content, giving it a zero height footprint (with no margins), that would be the only thing to look for, the non floated thing pushing things down.

Reply:
Thanks @jasmussen ✋ I've removed the float completely now.